### ISSUE TITLE
Is this project still running?

### ISSUE THREAD

**Issue by Kianytty** [wnielson/Plex-Remote-Transcoder#74]

**Comment by liviynz** [comment]
Yes, just currently no active new development but could change soon.

**Comment by Kianytty** [comment]
I look forward to it

**Comment by prinzpi** [comment]
Any updates regarding this project?
Would be exactly what I need. 👍

**Comment by killermosi** [comment]
I'm waiting for #53 to be merged.

**Comment by sachk** [comment]
https://github.com/UnicornTranscoder/UnicornTranscoder works pretty well currently

**Comment by killermosi** [comment]
@sachk: Thanks, I'll check it out, although it does not seem to be able to do distributed transcoding (splitting a single stream transcoding job between multiple slaves).
